autoscale/src/main/java/com/sequenceiq/periscope/monitor/client/YarnMetricsClient.java

```diff
@@ -65,7 +70,10 @@ public YarnScalingServiceV1Response getYarnMetricsForCluster(Cluster cluster, St
                 tlsConfig.getClientCert(), tlsConfig.getClientKey(), true);
         String yarnApiUrl = String.format(YARN_API_URL, clusterProxyUrl, cluster.getStackCrn());
 
+        long start = Instant.now().toEpochMilli();
```

Comment:
Use Clock here as well - similar to what is done in YarnBasedScalingAdjustmentService ?